### ISSUE TITLE
vrmonitor.exe keeps crashing

### ISSUE THREAD

**Issue by keem85** [mbucchia/XR_APILAYER_NOVENDOR_nis_scaler#13]
This nis scaler keeps restarting vrmonitor, and it results in freeze ingame after a while.. It can happen after 10 minutes, or it can go as long as 20 minutes.. Then everything freezes and I have to force kill Oculus runtime and SteamVR..
Is there any way I can uninstall this software temporarily? I can't figure out where the uninstaller is

I'm using a Quest 2 with latest Windows 11, tested with several nvidia drivers
i7 12900k and an RTX 9030 Rog Strix

**Comment by mbucchia** [comment]
I don't think this vrmonitor/vrserver has anything to do with OpenXR. So I doubt my application is causing this. 

Anyway to answer your questions you may use Windows' Add or remove programs (from the Start menu), then look for OpenXR-NIS-Scaler.

**Comment by keem85** [comment]
Hmm, are you sure? I started getting this problem after trying this progrem (which is fantastic by the way). On initial setup via the GUI, the vrmonitor.exe keeps restarting constantly whilst having your GUi open.. Once I close down GUI, the constant start/stop of vrmonitor.exe stops. I thought that this was a strong indication of this app causing it, but I don't know for sure.

**Comment by mbucchia** [comment]
Interesting. I am not familiar with vrmonitor (or generally with Oculus and Steam), so I was under the impression this did not involve OpenXR. I could see one reason why this would happen while the configuration tool is running.

However, I will not fix this **because the configuration tool will be removed and replaced by in-game display in the future**. So this issue will disappear by nature.

In the meantime, I hope you can live with this issue. To be clear, you do not need to keep the configuration tool open once you are done with your settings. So hopefully this is inconvenient right now, but should not block you from using the upscaler in your games.

Again the next version will not have this problem. Thanks for taking the time to report this!
